organisations/tests/test_models.py

```diff
@@ -127,3 +130,68 @@ def test_user_str():
         email="",
     )
     assert user_without_name_and_email.__str__() == user_without_name_and_email.username
+
+
+@pytest.mark.django_db
+def test_too_old_personal_data():
+    too_old = timezone.now() - relativedelta(months=24, days=1)
+    still_valid = timezone.now() - relativedelta(months=24, days=-1)
+
+    # ### EXPECTED PERSON OBJECTS IN RESULTS
+
+    expected_people = PersonFactory.create_batch(5, user=None)
+
+    # person with a too old enrolment
+    EnrolmentFactory(person=expected_people[0], occurrence__end_time=too_old)
+
+    # person with a too old study group
+    EnrolmentFactory(person=expected_people[1], occurrence__end_time=too_old)
+
+    # person with a too old enrolment and a too old study group
+    EnrolmentFactory(person=expected_people[2], occurrence__end_time=too_old)
+    EnrolmentFactory(
+        study_group__person=expected_people[2], occurrence__end_time=too_old
+    )
```

Comment:
Is this even a possible scenario? Isn't it a whole new person then? Or is it possible when it's a multi-occurrence enrolment?

Reply:
Probably not in practice 🙂. Removed.